docs/StrategyFormatChangelog.md

```diff
@@ -1,8 +1,17 @@
 # Změny formátu strategie
 
+## Robozonky 5.4.0
+
+Po zavedení nové evropské směrnice [PSD2](https://cs.wikipedia.org/wiki/Sm%C4%9Brnice_PSD2) odstranilo 
```

Comment:
@triceo please check this user facing message (it's the markdown document linked from the warning shown above). Do you consider this understandable for end users?

Reply:
This is very clear. However, the screenshot contains a typo - "nastavenou" instead of "nastaveno".